### DISCUSSION TITLE
test installation of npm

### DISCUSSION THREAD

**Discussion by rootsmusic** [coreybutler/nvm-windows#819] [Support] [answered]
I don't know Bash, nvm, node, npm and Git.  I installed nvmw using nvm-setup.exe, and I executed `nvm install latest`.

However, I'm getting errors when I run npm.  How can I test whether npm is fully installed?  Thanks.

VERSIONS
Desktop: Windows 10
Git: 2.36.1.windows.1
nvm for Windows: 1.1.9
Node.js: 18.3.0
npm: 8.11.0

**Comment by adestr** [comment, marked as answer]
Have you also run `nvm use latest`? Installing a version and setting it as the active version are two separate steps.
